### ISSUE TITLE
Exception: One of the initial species must be the solvent with the same string name

### ISSUE THREAD

**Issue by nickvandewiele** [ReactionMechanismGenerator/RMG-Py#693]
When running the current master of RMG-Py with v1.1.0 of RMG-database with the "solvent_hexane" simulation case:

```python
database(
    thermoLibraries = ['primaryThermoLibrary'],
    reactionLibraries = [],
    seedMechanisms = [],
    kineticsDepositories = 'default',
    kineticsFamilies = 'default',
    kineticsEstimator = 'rate rules',
)

species(
    label = "oxygen",
    reactive = True,
    structure = adjacencyList(
"""
multiplicity 3
1 O u1 p2 c0 {2,S}
2 O u1 p2 c0 {1,S}
"""),
)

species(
    label = "hexane",
    reactive = True,
    structure = adjacencyList(
"""
1  C u0 p0 c0 {2,S} {7,S} {8,S} {9,S}
2  C u0 p0 c0 {1,S} {3,S} {10,S} {11,S}
3  C u0 p0 c0 {2,S} {4,S} {12,S} {13,S}
4  C u0 p0 c0 {3,S} {5,S} {14,S} {15,S}
5  C u0 p0 c0 {4,S} {6,S} {16,S} {17,S}
6  C u0 p0 c0 {5,S} {18,S} {19,S} {20,S}
7  H u0 p0 c0 {1,S}
8  H u0 p0 c0 {1,S}
9  H u0 p0 c0 {1,S}
10 H u0 p0 c0 {2,S}
11 H u0 p0 c0 {2,S}
12 H u0 p0 c0 {3,S}
13 H u0 p0 c0 {3,S}
14 H u0 p0 c0 {4,S}
15 H u0 p0 c0 {4,S}
16 H u0 p0 c0 {5,S}
17 H u0 p0 c0 {5,S}
18 H u0 p0 c0 {6,S}
19 H u0 p0 c0 {6,S}
20 H u0 p0 c0 {6,S}
"""),
)

liquidReactor(
    temperature = (400,"K"),
    initialConcentrations={
        "hexane": (7.598e-3,'mol/cm^3'),
        "oxygen": (4.953e-6,'mol/cm^3'),
    },
    terminationTime = (10,"s"),
)

solvation(
    solvent='hexane'
)

simulator(
    atol = 1e-16,
    rtol = 1e-08,
)

model(
    toleranceMoveToCore = 0.001,
    toleranceKeepInEdge = 1e-9,
    toleranceInterruptSimulation = 0.1,
    maximumEdgeSpecies = 100000,
    minCoreSizeForPrune = 50,
    minSpeciesExistIterationsForPrune = 2,
    filterReactions = 0,
)

options(
    units = "si",
    saveRestartPeriod = None,
    generateOutputHTML = False,
    generatePlots = False,
    saveSimulationProfiles = False,
    saveEdgeSpecies = True,
    verboseComments = False,
)
```

I get the following error:

```python
One of the initial species must be the solvent with the same string name
Traceback (most recent call last):
  File "rmg.py", line 164, in <module>
    rmg.execute(**kwargs)
  File "/vagrant_data/RMG-Py/rmgpy/rmg/main.py", line 524, in execute
    self.initialize(**kwargs)
  File "/vagrant_data/RMG-Py/rmgpy/rmg/main.py", line 442, in initialize
    self.database.solvation.checkSolventinInitialSpecies(self,Species.solventStructure)
  File "/vagrant_data/RMG-Py/rmgpy/data/solvation.py", line 948, in checkSolventinInitialSpecies
    raise Exception('One of the initial species must be the solvent with the same string name')
Exception: One of the initial species must be the solvent with the same string name
make: *** [solvent] Error 1

```

**Comment by yunsiechung** [comment]
I found the cause of the error. It was the indentation mistake that I made in rmgpy/data/solvation.py

Originally, it was:
```
def checkSolventinInitialSpecies(self,rmg,solventStructure):
        for spec in rmg.initialSpecies:
            if solventStructure is not None:
                spec.isSolvent = spec.isIsomorphic(solventStructure)
            else:
                spec.isSolvent = rmg.solvent == spec.label
            if not any([spec.isSolvent for spec in rmg.initialSpecies]):
                if solventStructure is not None:
                    logging.info('One of the initial species must be the solvent')
                    raise Exception('One of the initial species must be the solvent')
                else:
                    logging.info('One of the initial species must be the solvent with the same string name')
                    raise Exception('One of the initial species must be the solvent with the same string name')
```

But it should have been this:
```
def checkSolventinInitialSpecies(self,rmg,solventStructure):
        for spec in rmg.initialSpecies:
            if solventStructure is not None:
                spec.isSolvent = spec.isIsomorphic(solventStructure)
            else:
                spec.isSolvent = rmg.solvent == spec.label
        if not any([spec.isSolvent for spec in rmg.initialSpecies]):
            if solventStructure is not None:
                logging.info('One of the initial species must be the solvent')
                raise Exception('One of the initial species must be the solvent')
            else:
                logging.info('One of the initial species must be the solvent with the same string name')
                raise Exception('One of the initial species must be the solvent with the same string name')
```

Previously, my code was checking whether the first initial species was the solvent. 
With the correction, the code checks whether Any of the initial species is the solvent.

Sorry for the error!

**Comment by nickvandewiele** [comment]
closed by #694
